content/docs/tutorials/getting-started-aws-letsencrypt/README.md

```diff
@@ -0,0 +1,654 @@
+---
+title: Deploy cert-manager on Amazon Elastic Kubernetes (EKS) and use Let's Encrypt to sign a certificate for an HTTPS website
+description: |
+    Learn how to deploy cert-manager on Amazon Elastic Kubernetes Service (EKS)
+    and configure it to get a signed certificate from Let's Encrypt for an HTTPS web server,
+    using the DNS-01 protocol and Amazon Route53 DNS with workload identity federation.
+---
+
+*Last Verified: 3 September 2024*
+
+In this tutorial you will learn how to deploy and configure cert-manager on Amazon Elastic Kubernetes Service (EKS)
+and how to deploy an HTTPS web server and make it available on the Internet.
+You will learn how to configure cert-manager to get a signed certificate from Let's Encrypt,
+which will allow clients to connect to your HTTPS website securely.
+You will configure cert-manager to use the [Let's Encrypt DNS-01 challenge protocol](https://letsencrypt.org/docs/challenge-types/#dns-01-challenge) with Amazon Route53 DNS,
+using IAM Roles for Service Accounts (IRSA) to authenticate to Amazon Route53.
+
+> **Amazon Web Services**: A suite of cloud computing services by Amazon.<br/>
+> **Kubernetes**: Runs on your servers. Automates the deployment, scaling, and management of containerized applications.<br/>
+> **cert-manager**: Runs in Kubernetes. Obtains TLS / SSL certificates and ensures the certificates are valid and up-to-date.<br/>
+> **Let’s Encrypt**: An Internet service. Allows you to generate free short-lived SSL certificates.
+
+# Part 1
+
+In the first part of this tutorial you will learn the basics required to deploy an HTTPS website on an Amazon Elastic Kubernetes Service (EKS) cluster, using cert-manager to create the SSL certificate for the web server.
+You will create a DNS domain for your website, create an EKS cluster, install cert-manager, create an SSL certificate and then deploy a web server which responds to HTTPS requests from clients on the Internet.
+But the SSL certificate in part 1 is only for testing purposes.
+
+In part 2 you will learn how to configure cert-manager to use Let's Encrypt and Route53 DNS to create a trusted SSL certificate which you can use in production.
```

Comment:
suggestion (non-blocking): The line "But the SSL certificate in part 1 is only for testing purposes." doesn't really flow with the rest of the text IMO. How about a minor tweak?

```suggestion
In the first part of this tutorial you will learn the basics required to deploy an HTTPS website on an Amazon Elastic Kubernetes Service (EKS) cluster, using cert-manager to create the SSL certificate for the web server.

You will create a DNS domain for your website, create an EKS cluster, install cert-manager, create an SSL certificate and then deploy a web server which responds to HTTPS requests from clients on the Internet.

The SSL certificate in part 1 is only for testing purposes; in part 2 you will learn how to configure cert-manager to use Let's Encrypt and Route53 DNS to create a trusted SSL certificate which you can use in production.
```